docs/json-api/neu_json_plugin.json

```diff
@@ -1,19 +1,10 @@
 {
     "add_plugin_req": {
-        "kind": 1,
-        "node_type": 1,
-        "name": "plugin_name",
         "lib_name": "plugin_name.so"
     },
     "del_plugin_req": {
         "id": 1
     },
-    "update_plugin_req": {
```

Comment:
Don't delete update_plugin_req, we need to update plugin when user will to update new plugin library.